dist/profile/manifests/jiraveg.pp

```diff
@@ -0,0 +1,32 @@
+# profile for jira
+class profile::jiraveg (
+  $jira_javahome,
+  $jira_version
+){
+  $pkgs = ['openjdk-7-jre-headless', 'facter']
```

Comment:
I also think so. The only thing i found that facter tool need exist so jira can put file in /etc/facter/ directory. Maybe it can be a part of base installation.